x-pack/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -81,6 +88,10 @@ export function AlertDetails() {
     },
   ]);
 
+  const onAlertStatusChange = () => {
```

Comment:
The name is a bit confusing because here, we only set the status to `untracked`. How about changing the name to something like `onUntrackAlert`?